dvc/parsing/interpolate.py

```diff
@@ -98,12 +98,12 @@ def _(obj: dict):
                     result += f"--no-{k} "
 
         elif isinstance(v, str):
-            result += f"--{k} '{v}' "
+            result += f'--{k} "{v}" '
```

Comment:
What happens if the value has a mixed quotes? Do we handle that or escape in any way?

Reply:
I escaped double quotes and added a `mixed_quote_string`.